SpatialGDK/Source/SpatialGDK/SpatialGDK.Build.cs

```diff
@@ -41,7 +41,9 @@ public SpatialGDK(ReadOnlyTargetRules Target) : base(Target)
                 "OnlineSubsystemUtils",
                 "Projects",
                 "ReplicationGraph",
-                "Sockets"
+                "Sockets",
+                "Slate",
```

Comment:
Need both slate and umg?

Reply:
UMG for widget types, Slate for input types when binding the toggle key.